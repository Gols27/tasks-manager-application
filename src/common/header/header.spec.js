import React from 'react';
import Header from './';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';


it('renders without crashing', () => {
  const wrapper = shallow(<Header />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
