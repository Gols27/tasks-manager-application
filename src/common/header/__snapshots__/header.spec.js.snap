// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="header-container"
>
  <img
    alt="logo"
    className="header-logo"
    src="application-logo.png"
  />
  <div
    className="header-text"
  >
    TASKS APPLICATION
  </div>
</div>
`;
